### ISSUE TITLE
Statistics don't open from CLI

### ISSUE THREAD

**Issue by WBTMagnum** [projecthamster/hamster#463]
On the command line there is the possibility to start dedicated hamster windows. E.g. help states: "overview / statistics / about: launch specific window".

When trying to open the statistics window, the following error is thrown:
```bash
$ hamster statistics
Traceback (most recent call last):
  File "/usr/bin/hamster", line 418, in <module>
    getattr(hamster_client, command)(*args.action_args)
  File "/usr/bin/hamster", line 160, in statistics
    self._launch_window("statistics")
  File "/usr/bin/hamster", line 148, in _launch_window
    getattr(server, window_name)()
  File "/usr/lib/python3/dist-packages/dbus/proxies.py", line 70, in __call__
    return self._proxy_method(*args, **keywords)
  File "/usr/lib/python3/dist-packages/dbus/proxies.py", line 145, in __call__
    **keywords)
  File "/usr/lib/python3/dist-packages/dbus/connection.py", line 651, in call_blocking
    message, timeout)
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.UnknownMethod: Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/dbus/service.py", line 654, in _message_cb
    (candidate_method, parent_method) = _method_lookup(self, method_name, interface_name)
  File "/usr/lib/python3/dist-packages/dbus/service.py", line 248, in _method_lookup
    raise UnknownMethodException('%s is not a valid method' % method_name)
dbus.exceptions.UnknownMethodException: org.freedesktop.DBus.Error.UnknownMethod: Unknown method: statistics is not a valid method
```
Launching `overview` ~~and `about`~~ window works as expected. Is there even a `statistics` window in the revamped client? `about` window also stopped working now.

**Just for the Record:**
* Hamster was build from updated GIT repo today (revision a934effe971c72777cf494f5fde9cd58a0ac0d7c).
* OS: Ubuntu 18.04.3 LTS

**Comment by GeraldJansen** [comment]
As you guessed, there is no separate `statistics` window in the revamped client. Program hamster-cli should be adjusted, including the note in --help. Instead, the `hamster add`  option to open the edit activity window directly should be added to the  documentation.

**Comment by WBTMagnum** [comment]
Thanks for the swift reply and confirmation.

Since `about` also stopped working, I just updated the description above.

**Comment by GeraldJansen** [comment]
Can this be closed now that PR #480 has been merged? Note that `hamster about` seems to work now.
